### PR TITLE
New messages: SalesReports_DisplayDailyOrSummarizedReport and SalesReports_DisplayNetRemitReport

### PR DESCRIPTION
**Pull request by therealartz**

### CODE REVIEW

File: docs/samples.rst [hunk 1]
```diff
@@ -3684,4 +3688,39 @@ Request a sales report from a certain date to another date, issued in all office
 
     $salesReportResult = $client->salesReportsDisplayQueryReport($opt);
 
+-------------------------
```
Comment (DerMika) [inline docs/samples.rst]: AFAIK in RST your lines above and below titles need to be of the exact same length of the text.

See http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#sections

File: src/Amadeus/Client/Struct/SalesReports/DisplayDailyOrSummarizedReport.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    protected function loadCurrency($type, $currency)
+    {
+        if ($this->checkAnyNotEmpty($type, $currency)) {
+            $this->currency = new CurrencyInfo($type, $currency);
+        }
+    }
+
+    protected function loadSalesReportIdentification($number, $type)
+    {
+        if (!empty($number) && !empty($type)) {
```
Comment (DerMika) [inline src/Amadeus/Client/Struct/SalesReports/DisplayDailyOrSummarizedReport.php]: For this I made the helper function `checkAllNotEmpty()`

**Comment by DerMika** [comment]
I will merge when you've fixed the 2 issues I mentioned.

File: src/Amadeus/Client/Struct/SalesReports/DisplayDailyOrSummarizedReport.php (hunk 2 above)
Comment (therealartz) [reply src/Amadeus/Client/Struct/SalesReports/DisplayDailyOrSummarizedReport.php]: Oh, indeed, I somehow didn't notice, although it's used directly in method above this :fearful:

File: tests/Amadeus/Client/Struct/SalesReports/DisplayDailyOrSummarizedReportTest.php [hunk 3]
```diff
@@ -0,0 +1,71 @@
+<?php
```
[the remark this answers is not in the document]
Comment (therealartz) [reply tests/Amadeus/Client/Struct/SalesReports/DisplayDailyOrSummarizedReportTest.php]: I just copy-pasted `DisplayQueryReportTest` that also doesn't have this header

File: src/Amadeus/Client/Struct/SalesReports/DisplayDailyOrSummarizedReport.php (hunk 2 above)
Comment (DerMika) [reply src/Amadeus/Client/Struct/SalesReports/DisplayDailyOrSummarizedReport.php]: Sure you're usng the correct function? Your previous code checked if both were not empty, this function checks if at least one isn't empty. You probably want `checkAllNotEmpty()`

**Comment by DerMika** [comment]
Thanks for your contributions!

**Pull request merged**